proxy/src/main.rs

```diff
@@ -105,7 +110,7 @@ async fn handle_stream_response(resp: Response) -> Result<()> {
 
 async fn proxy() -> Result<()> {
     // Get the hostname from the environment
-    let origin = read(ENV_CONFIG);
+    let origin = read(ENV_CONFIG).expect("No origin");
```

Comment:
Based on the usage, we should probably change `read()`'s signature to return Result<String> instead of Option, so then we can use ? to bubble the error up.

Reply:
Nice.  Done in 11e339d763d632783d35d40a8d706d58310f4b14.